src/metatensor/models/cli/train.py

```diff
@@ -383,6 +383,17 @@ def _train_model_hydra(options: DictConfig) -> None:
     ###########################
     logger.info("Setting up model")
 
+    # TODO: make gradients more flexible and explicit in the extraction
```

Comment:
I don't understand what this comment means. You want to allow the user to specify which gradients to compute?

Reply:
With that comment I mean that a direct way to look up the gradients would be to get them from the configuration dict of the training run, which would probably lead to more readable code. The two are equivalent however

Reply:
could you update the comment to say something like your answer? So the next people looking at it understand what's going on